js/ui/overlay/ui.overlay.js

```diff
@@ -257,9 +259,15 @@ const Overlay = Widget.inherit({
         return this._$content;
     },
 
-    _init: function() {
+    _setDeprecatedOptions() {
         this.callBase();
+        extend(this._deprecatedOptions, {
+            'elementAttr': { since: '21.2', message: 'Use the "wrapperAttr" option instead' }
```

Comment:
```suggestion
            'elementAttr': { since: '21.2', alias: 'wrapperAttr' }
```
https://github.com/DevExpress/DevExtreme/blob/ec0da494a16f85c8e7a3ada97e7c2269949c2909/js/core/component.js#L133